srcalc/src/main/webapp/WEB-INF/views/admin/utilizationReportResults.jsp

```diff
@@ -0,0 +1,101 @@
+<%@ taglib uri="/WEB-INF/srcalc.tld" prefix="srcalc" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt" %>
+<%@ taglib uri="http://www.springframework.org/tags" prefix="spring" %>
+<%@ taglib uri="http://www.springframework.org/tags/form" prefix="form" %>
+<%@ taglib uri="http://www.joda.org/joda/time/tags" prefix="joda" %>
+
+<srcalc:adminPage title="Utilization Report">
+
+<section>
+    <h2>ASRC Utilization Report - ${report.generationDate}</h2>
+    
+    <p>
+    The below tables may be copied and pasted into Excel.
+    </p>
+    
+    <c:if test="${report.results.truncated}">
+    <p class="error">
+    Your search returned more results than could be displayed. Please refine your search.
+    </p>
+    </c:if>
+    
+    <h3>Summary Data</h3>
+    
+    <table class="reportTable">
+    <thead>
+    <tr>
+        <th>Specialty</th>
+        <th>Total</th>
+        <th>Signed</th>
+        <%-- Set the width to make sure these don't get too large. --%>
+        <th style="width: 7em;">Time to First Run Average (Minutes)</th>
+        <th style="width: 7em;">Time to Sign Average (Minutes)</th>
```

Comment:
Because I was in a rush and couldn't think of a maintainable way to put it in the CSS. But I thought of one. :)